doc/source/cluster/cluster_under_construction/getting-started.rst

```diff
@@ -26,13 +15,17 @@ One of Ray's strengths is the ability to leverage multiple machines for
 distributed execution. Ray can, of course, be run on a single machine (and is
 done so often), but the real power is using Ray on a cluster of machines.
```

Comment:
```suggestion
distributed execution. Ray can be run on a single machine and is
done so often, but the real power of Ray is the ability to seamlessly scale to a cluster of machines.
```

Reply:
This is still ever-so-slightly ungrammatical.
How about:
`Ray is great for multiprocessing on a single machine. However, the real power of Ray is the ability to seamlessly scale to a cluster of machines.`

Reply:
good catch... didn't read that paren phrase well